src/Audio_details.md

```diff
@@ -0,0 +1,192 @@
+Note that the envelope functionality changes the volume, but not the value stored in NRx2, and thus doesn't disable the DACs.
+
+If a DAC is enabled, the digital range $0 to $F is lineraly translated to the analog range -1 to 1, in arbitrary units.
+Importantly, the slope is negative: "digital 0" maps to "analog 1", not "analog -1".
+
+If a DAC is disabled, it fades to an analog value of 0.
+The nature of this fade is not entirely deterministic and varies between models.
+
+NR52's low 4 bits report whether the channels are turned on, not their DACs.
+
+### Channels
+
+A channel is activated by a write to NRx4's MSB, unless its DAC is off, which forces it to be disabled as well.
+The opposite is not true, however: a disabled channel outputs 0, which an enabled DAC will dutifully convert into "analog 1".
+
+A channel can be deactivated in one of the following ways:
+- Turning off its DAC
+- Its [length timer](<#Length timer>) expiring
+- (CH1 only) [Frequency sweep](<#FF10 — NR10: Channel 1 sweep>) overflowing the frequency
+
+### Pulse channels (CH1, CH2)
+
+Each pulse channel has an internal "duty step" counter, which is used to index into [the selected waveform](<#FF11 — NR11: Channel 1 length timer & duty cycle>) (each background stripe corresponds to one "duty step")[^pulse_lut].
+The "duty step" increments at 8 times [the channel's frequency](<#FF13 — NR13: Channel 1 wavelength low \[write-only\]>)).
+
+The "duty step" counter cannot be reset, except by turning the APU off, which sets both back to 0.
+Retriggering a pulse channel causes its "duty step timer" to reset, thus retriggering a pulse channel often enough will cause its "duty step" to never advance.
+
+When first starting up a pulse channel, it will *always* output a (digital) zero.
+
+[^pulse_lut]:
+Actually, there is not LUT, but the manipulations done to the counter's bits are equivalent.
+
+### Wave channel (CH3)
+
+CH3 has an internal "sample index" counter.
+Each time it is ticked (as determined by the "wavelength" in NR33/NR34), that "sample index" is incremented, and then the corresponding "sample" (nibble) is read from wave RAM.
+(This means that sample #0 is skipped when first starting up CH3.)
+
+CH3 does not emit samples directly, but stores every sample read into a buffer, and emits that continuously; (re)triggering the channel does *not* clear nor refresh this buffer, so the last sample ever read will be emitted again.
+This buffer *is* cleared when turning the APU on, so CH3 will emit a "digital 0" when first powered on.
+
+CH3 output level control does not, in fact, alter the output level.
+It shifts the **digital** value CH3 is outputting, not the analog value.
+This only matters when changing the setting mid-playback: the digital values being shifted bias them towards 0, which biases the analog output towards "1"; the HPF will smooth this over time, but not instantly.
```

Comment:
I'm not super sure about how we are using `bias` as a verb here. We should try to find something more technical.

Reply:
I believe "to bias" is a verb used fairly often in signal processing.